langchain/src/schema/output_parser.ts

```diff
@@ -116,6 +116,70 @@ export class StringOutputParser extends BaseOutputParser<string> {
   getFormatInstructions(): string {
     return "";
   }
+
+  async *_transform(
+    inputGenerator: AsyncGenerator<string | BaseMessage>
+  ): AsyncGenerator<string> {
+    for await (const chunk of inputGenerator) {
+      if (typeof chunk === "string") {
+        yield this.parseResult([{ text: chunk }]);
+      } else {
+        yield this.parseResult([{ message: chunk, text: chunk.content }]);
+      }
+    }
+  }
+
+  async *transform(
+    inputGenerator: AsyncGenerator<string | BaseMessage>,
+    options: BaseCallbackConfig
+  ): AsyncGenerator<string> {
+    yield* this._streamWithConfig(this._transform(inputGenerator), {
+      ...options,
+      runType: "parser",
+    });
+  }
+}
+
+/**
+ * OutputParser that parses LLMResult into the top likely string and
+ * encodes it into bytes.
+ */
+export class EncodingOutputParser extends BaseOutputParser<Uint8Array> {
```

Comment:
Might we want to call this BytesOutputParser or TextEncoderOutputParser

Reply:
It accepts chat chunks so I think BytesOutputParser is better